### DISCUSSION TITLE
can you help me please?

### DISCUSSION THREAD

**Discussion by PamelaTriutami7845** [datasciencecompleteguide/Machine-Learning#35] [General]
error:
cv2.error: OpenCV(4.5.5) :-1: error: (-5:Bad argument) in function 'resize'
> Overload resolution failed:
>  - src is not a numpy array, neither a scalar
>  - Expected Ptr<cv::UMat> for argument 'src'

mycode:

import cv2

def empty():
    pass

path = "../images/images.jfif"

cv2.namedWindow("TrackBars")
cv2.resize("TrackBars",640,240)
cv2.createTrackbar("Hue min","TrackBars",0,179,empty)
img = cv2.imread(path)

imgHSV = cv2.cvtColor(img,cv2.COLOR_BGR2HSV)

cv2.imshow("original", img)
cv2.imshow("imagesHSV", imgHSV)
cv2.waitKey(0)

**Comment by datasciencecompleteguide** [comment]
path = "../images/images.jfif"
Give the exact and complete location like "e://some_folder//image.jfif" (Don't use this "../../" notation)

cv2.resize("TrackBars", (640, 240))
Size in the resize function will be passed as the tuple

Rest of the things are right!!
